### PR TITLE
Fix build on Illumos

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: src/auth/basic/LDAP/basic_ldap_auth.cc [hunk 1]
```diff
@@ -134,6 +134,10 @@ PFldap_start_tls_s Win32_ldap_start_tls_s;
 #include <lber.h>
 #include <ldap.h>
 
+#if _SQUID_SOLARIS_
```
Comment (rousskov) [inline src/auth/basic/LDAP/basic_ldap_auth.cc:137]: Should we generalize this to other "broken" platforms?

```suggestion
#ifndef LDAP_SECURITY_ERROR
```

**Review by rousskov — COMMENTED** [review]
It would be nice to cite the actual build errors in the PR description so that others facing these errors can easily find this fix. If you have those compiler errors saved, please paste them there.

We can help with Markdown/commit [formatting](https://github.com/measurement-factory/anubis/blob/master/README.md#commit-message) if you would rather not deal with 72-character-lines ASCII-only restrictions and four-leading-spaces Markdown quoting.

File: src/auth/basic/LDAP/basic_ldap_auth.cc (hunk 1 above)
Comment (devnexen) [reply src/auth/basic/LDAP/basic_ldap_auth.cc:137]: this is the only unix I stumbled across this but not against the change.

**Comment by devnexen** [comment]
errors are long gone + I forgot if I compiled with clang or gcc, but it s for sure not previous declared function and undefined reference kind of errors.

**Comment by rousskov** [comment]
I thought you have access to the build environment where you can easily reproduce the old errors and retest the latest fixes. If not, that is OK -- we do not _have_ to provide those errors in the commit message.

**Comment by rousskov** [comment]
OK to test

**Review by rousskov — APPROVED** [review]

**Review by yadij — APPROVED** [review]

**Pull request closed without merging**